src/styles/common/flex-box.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from 'styled-components';
+
+interface Flexbox {
+  direction?: string;
+  justify?: string;
+  align?: string;
+  gap?: string;
+}
```

Comment:
```suggestion
interface Flexbox {
  direction?: CSSProperties['flexDirection'];
  justify?: CSSProperties['justifyContent'];
  align?: CSSProperties['alignItems'];
  gap?: CSSProperties['gap'];
}
```

이렇게 하면 좀 더 자동완성이 잘되어서 좋을거 같아요


그리고 Flex 까지 만드시다니 좋은거 같아요!!!